src/utilities/services/history-helper.js

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { encodedToken } from "utilities/token";
+/**
+ * add video in a History
+ * @param {*} video object contains video details
+ * @param {*} playListDispatch 
+ */
+const addVideoInHistory = async (video, playListDispatch) => {
+    try {
+        const { data } = await axios.post("/api/user/history", { video }, {
+            headers: { authorization: encodedToken }
+        })
+        playListDispatch({ type: "ADD_HISTORY", payload: data.history })
+    } catch (error) {
+        console.log(error);
```

Comment:
Instead of consoling the error, you can use `toastify` to show pop up whenever some error occurs.